Lexicon.md

```diff
@@ -5,10 +5,13 @@
 * __Vulcan Set__: The set of all __Bit Strings__ of infinite length $$ \mathbb{V} := \{ B_0^{\infty}, B_1^{\infty}, \dots, B_{\infty}^{\infty} \} , \ B_i^{\infty} \neq B_j^{\infty} $$.
 * __Information__: Anything that can be encoded into a __Bit String__.
 * __Environment__: __Information__ that is defined relative to a particular __Agent__. It is seperated from an __Agent__ by an __Agent Boundary__, and encompasses all content needed to process any relevant __Actions__ and __Observations__.
+* __World__: Synonymous with __Environment__.
```

Comment:
I think of World as the subset of the Environment that an Agent cares about. I don't have a strong opinion if you don't think that is an improvement,

Reply:
We could flag this for discussion at the next meeting, Ill merge as is for now though and we can revisit this.